ob_intern/Controller/Create Short Video/CreateShortVideoViewController.swift

```diff
@@ -55,25 +65,81 @@ class CreateShortVideoViewController: UIViewController {
         textViewThoughts.textColor = UIColor.lightGray
         imageViewVideo.layer.cornerRadius = 16
         imageViewVideo.clipsToBounds = true
-        imageViewVideo.sd_setImage(with: URL(string: "https://images.unsplash.com/photo-1609171712489-45b6ba7051a4?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8M3x8c3Vuc2V0JTIwYWVzdGhldGljfGVufDB8fDB8fHww&auto=format&fit=crop&w=800&q=60"))
+        imageViewVideo.sd_setImage(with: URL(string: "https://images3.alphacoders.com/110/1108129.jpg"))
         labelChooseFrontCover.layer.cornerRadius = 12
         labelChooseFrontCover.clipsToBounds = true
         self.navigationController?.setNavigationBarHidden(true, animated: false)
+        setButtonImage(imageName: "chevron.left",
+                       iconColor: .black,
+                       button: buttonBack)
+        buttonPost.setTitle("โพสต์", for: .normal)
 
+        buttonPost.addSubview(spinner)
+        spinner.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            spinner.centerXAnchor.constraint(equalTo: buttonPost.centerXAnchor),
+            spinner.centerYAnchor.constraint(equalTo: buttonPost.centerYAnchor)
+        ])
+        dismissEditing()
      
     }
     
+    func updateLoadingIndicator() {
+        if isLoading {
+            spinner.startAnimating()
+            buttonPost.isEnabled = false
+            buttonPost.setTitle(" ", for: .disabled)
```

Comment:
```suggestion
            buttonPost.setTitle("", for: .disabled)
```
ย้ายไปตรง setup view ได้เลยครับ